src/Plugin.php

```diff
@@ -55,6 +73,9 @@ public function __construct(array $config = [])
             throw new \InvalidArgumentException("Missing required configuration key 'nicks'");
         }
         $this->setNicks($config['nicks']);
+        if (!empty($config['recovery'])) {
```

Comment:
:smiley_cat: 

But yes, the property is initialised to false, and is *changed* to true if and only if `$config['recovery']` exists and is not equivalent to boolean false.